Resources/Textures/_RMC14/Objects/Misc/urn.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

4.0 is heavily prefered
```suggestion
  "license": "CC-BY-SA-4.0",
```